2019Labs/CustomSecurityContent/documentation/lab5_oval.adoc

```diff
@@ -158,6 +158,14 @@ We have the leading `definition` element:
 The `definition` specifies a `criteria` element.
 Here is a close-up of those critera:
 
+----
+    ...
+    <criteria operator="OR">
+      <criterion comment="TMOUT value in /etc/profile >= var_accounts_tmout"        test_ref="test_etc_profile_tmout" />
+      <criterion comment="TMOUT value in /etc/profile.d/*.sh >= var_accounts_tmout" test_ref="test_etc_profiled_tmout" />
+    </criteria>
+  </definition>
+  ...
```

Comment:
@matejak The snippet is duplicated. And from this section many paragraphs are in verbatim mode.

Reply:
Thanks for noticing, I have fixed that.